### DISCUSSION TITLE
Wysiwyg content Text too long

### DISCUSSION THREAD

**Discussion by cercec** [directus/directus#13211] [Archive]
On my Directus 9, I have a WYSIWYG field which has a TEXT type in my SQL database, and I need to add content that is more than 10000 caracters. I fixed it by replacing the type to LONGTEXT directly in the database, but is there a cleaner way to do that ? Is there a risk to do it manually ?

### To Reproduce

Put more than 8000 caracters in a Wysiwyg fiel

### Errors Shown

<img width="618" alt="image" src="https://user-images.githubusercontent.com/60812978/167677277-db560f0a-f5b6-4a9c-8674-c9f27b76e538.png">

### What version of Directus are you using?

9

### What version of Node.js are you using?

14.19.1

### What database are you using?

MySQL

### What browser are you using?

Chrome

### How are you deploying Directus?

locally

**Comment by rijkvanzanten** [comment]
What database are you using?

**Reply by cercec** [reply]
MySQL 8.0 :)

**Reply by rijkvanzanten** [reply]
You can safely change the type in the database directly 👍🏻 Directus will use knex's `.text()` method to generate those columns. In most db vendors, there isn't really a difference between "text" / "longtext" etc, so it defaults to `text`

**Reply by cercec** [reply]
Great ! Thank you for taking the time to help me @rijkvanzanten 😄 

Maybe it could interesting to precise this in the documentation ?

**Reply by rijkvanzanten** [reply]
(cc docs wizard @erondpowell)

**Reply by erondpowell** [reply]
@rijkvanzanten This seems like something we should link in via the dynamic Github comment system, instead of directly.
What do you think?

**Reply by rijkvanzanten** [reply]
It's definitely more of a "FAQ" sorta item than a full docs page, yeah
